Sources/MinderaDesignSystem/Components/Buttons/ButtonAppearance.swift

```diff
@@ -3,16 +3,16 @@ import SwiftUI
 struct ButtonAppearance {
     let lightMode: ButtonAppearanceColorScheme
     let darkMode: ButtonAppearanceColorScheme
-}
 
-struct ButtonAppearanceColorScheme {
-    let standard: ButtonAppearanceState
-    let pressed: ButtonAppearanceState
-    let disabled: ButtonAppearanceState
-}
+    struct ButtonAppearanceColorScheme {
```

Comment:
given this is nested now, it can simply be called `ColorScheme`, and the state can be just `State`, making the full type "path" a bit easier on the eyes